cmd/mqtt/main.go

```diff
@@ -35,35 +36,30 @@ const (
 	defLogLevel = "error"
 	envLogLevel = "MF_MQTT_ADAPTER_LOG_LEVEL"
 	// MQTT
-	defMQTTHost             = "0.0.0.0"
 	defMQTTPort             = "1883"
 	defMQTTTargetHost       = "0.0.0.0"
 	defMQTTTargetPort       = "1883"
-	defMQTTForwarderTimeout = "30" // in seconds
-
-	envMQTTHost             = "MF_MQTT_ADAPTER_MQTT_HOST"
+	defMQTTForwarderTimeout = "30s" // 30 seconds
 	envMQTTPort             = "MF_MQTT_ADAPTER_MQTT_PORT"
 	envMQTTTargetHost       = "MF_MQTT_ADAPTER_MQTT_TARGET_HOST"
 	envMQTTTargetPort       = "MF_MQTT_ADAPTER_MQTT_TARGET_PORT"
 	envMQTTForwarderTimeout = "MF_MQTT_ADAPTER_FORWARDER_TIMEOUT"
 	// HTTP
-	defHTTPHost       = "0.0.0.0"
 	defHTTPPort       = "8080"
 	defHTTPScheme     = "ws"
 	defHTTPTargetHost = "localhost"
 	defHTTPTargetPort = "8080"
 	defHTTPTargetPath = "/mqtt"
-	envHTTPHost       = "MF_MQTT_ADAPTER_WS_HOST"
 	envHTTPPort       = "MF_MQTT_ADAPTER_WS_PORT"
-	envHTTPScheme     = "MF_MQTT_ADAPTER_WS_SCHEMA"
+	envHTTPScheme     = "MF_MQTT_ADAPTER_WS_SCHEME"
```

Comment:
I think that this is unnecessary option in general and should always be hardcoded to `ws` in mProxy

Reply:
I agree. If we add TLS termination on the MQTT adapter side, we can update. At the moment, this is a useless option that we'll never change form its default value. I'm removing it.